### PR TITLE
Clarify predicted events privacy implication

### LINKED ISSUE #518: Ensure predicted events only use input from the current partition
From https://www.w3.org/TR/2024/WD-pointerevents3-20240318/#predicted-events:
> Some user agents have built-in algorithms which, after a series of confirmed pointer movements, can make a prediction (based on past points, and the speed/trajectory of the movement) what the position of future pointer movements may be.

If the prediction is based on the user's behavior on other sites, or on the same site across storage clearing, it could let sites join identities they shouldn't be able to join. Would it be possible to restrict the possible source data for generating predictions to the current [partition](https://w3ctag.github.io/privacy-principles/#dfn-partition)?

### PR DESCRIPTION
**Pull request by patrickhlauke**
* clarifies that the expectation is for the prediction to only use the past events from the current gesture/interaction
* adds a note in the privacy/security section to explain that it's the UAs' responsibility not to expose fingerprintable additional data through this

Closes https://github.com/w3c/pointerevents/issues/518

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/pointerevents/pull/527.html" title="Last updated on Nov 6, 2024, 5:29 PM UTC (05f30c0)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/pointerevents/527/82a6df8...05f30c0.html" title="Last updated on Nov 6, 2024, 5:29 PM UTC (05f30c0)">Diff</a>

### CODE REVIEW

**Review by smaug---- — APPROVED** [review]
Looks fine to me

File: index.html [hunk 1]
```diff
@@ -1538,6 +1538,7 @@ <h1>Security and privacy considerations</h1>
...
         <p>As mitigation, user agents may consider including the ability for users to disable access to particular sensor data (such as angle, tilt, pressure), and/or to make it available only after an explicit opt-in from the user.</p>
+        <p>This specification defines the method by which authors can access "predicted events". The specification does not, itself, define the algorithms that user agents should use for their prediction. The specification authors envisage the algorithms to only rely on preceding pointer events related to the current gesture that a user is performing. It is the responsibility of user agents to ensure that their specific implementation of a prediction algorithm does not rely on any additional data - such as the user's full interaction history across different sites - that could expose information that could be used to "fingerprint" and track a particular user.</p>
```
Comment (jyasskin) [inline index.html]: It's more than just fingerprinting: you might be able to learn concrete facts about the user using their history. That said, the mitigations are the same.

```suggestion
        <p>This specification defines the method by which authors can access "predicted events". The specification does not, itself, define the algorithms that user agents should use for their prediction. The specification authors envisage the algorithms to only rely on preceding pointer events related to the current gesture that a user is performing. It is the responsibility of user agents to ensure that their specific implementation of a prediction algorithm does not rely on any additional data - such as the user's full interaction history across different sites - that could reveal sensitive information about a user or be used to "fingerprint" and track them.</p>
```

**Review by jyasskin — APPROVED** [review]

**Comment by patrickhlauke** [comment]
thanks for the suggestion/clarification @jyasskin ... added that in. otherwise, does this look appropriate to you?

**Review by jyasskin — APPROVED** [review]
Yep, perfect. Thanks!

**Pull request merged**